path.go

```diff
@@ -74,6 +87,21 @@ func getFinalRecord(zone string, from int) (record, error) {
 		return rec, fmt.Errorf("could not parse record: %s", err)
 	}
 
+	if rec.Type == "path" {
```

Comment:
Any specific reason to add path code in addition to the resolver changes?

Reply:
I think I've messed up something when I rebased the branch.